_includes/insights-band.html

```diff
@@ -6,7 +6,7 @@ <h5>usually every Monday at 09:00 AM EST / 14:00 UTC / 15:00 CEST / 18:30 IST</h
     </div>
     <div class="width-12-12 width-12-12-m">
       <div class="iframe-container">
-      <iframe src="https://embed.restream.io/player/index.html?token=6b40eef71ed45dd4ece244b0f0ef3436" width="960" height="576" frameborder="0" allowfullscreen></iframe>
+      <iframe src="https://www.youtube.com/embed/playlist?list=PLsM3ZE5tGAVatO65JIxgskQh-OKoqM4F2" width="960" height="576" frameborder="0" allowfullscreen></iframe>
```

Comment:
that wont show the live video will it?

Reply:
I *think*  it would show the most recent in the Insights playlist. Not sure how you publish them